howtos/oem-guide.rst

```diff
@@ -0,0 +1,135 @@
+will be ignored.
+
+3. All pre-installs/re-distributions will involve only our
+`downloadable <https://rockstor.com/dls.html>`_ installer images.
+And then only when said installers include Stable Release Candidate 10 or later status 'rockstor' packages.
+With Stable release status (non RC) strongly preferred: when available.
+
+.. warning::
+
+    Apparent Rockstor installs not conforming to all of (1), (2), and (3) above,
+    will be deemed modified:
+    with all the consequent licensing ramifications regarding redistribution of modified code.
+    We digitally sign all released rpm packages,
+    and all distribution repositories that host them.
+    We can only reasonably respond to support/development requests/enquiries for software that we distribute.
+
+Practical considerations
+------------------------
+
+Our installer is end-user orientated, but OEM compatible, with caveats.
+During the very first boot of our installers: the systems `root` user password is configured.
+As such the end user, not a pre-installer, is the intended client.
+See: :ref:`installer_howto` for all the steps involved.
+In brief:
+
+- Select Installation Disk
+- Select Locale
+- Select Keyboard Layout
+- Rockstor “Built on openSUSE” installer License Agreement
+- Select Time Zone
+- Enter Desired root User Password (with a confirmation step there-after)
+
+.. warning::
+
+    Only once the `Rockstor bootstrapping tasks` has completed,
+    can the system then be safely shutdown/halted.
+    Enabling the remainder of the install to be completed on the intended final hardware.
+
+.. note::
+
+    The Web-UI setup stage, until it is completed, will remain 'pending',
+    irrespective of reboots.
+    Enabling target drive transfer to the intended final system & network.
+
+**In summary**: before the initial Web-UI setup,
+but after confirmation of the `OK Rockstor bootstrapping tasks`,
+the installation disk can be safely transferred (assuming a clean shutdown) to final hardware.
+Only during the :ref:`webui_setup` stage do we establish the intended system-unique **Appliance ID**.
+See :ref:`aboutapplianceid` for details.
+
+Pre-installers are strongly requested to ensure their distributed systems do not result in fake/non-unique Appliance IDs.
+Repeat Appliance ID installs will be refused support by the The Rockstor Project.
+
+.. note::
+    If hardware preferred by a pre-installer results in fake/non-unique Appliance IDs,
+    please consider enabling our accommodation by informing support.
+    The Rockstor Project development team can then extend our list of known non-unique product_uuids.
+
+Given all of the above, and our end-user accessible installer,
+we recommend that Rockstor bundling takes the form of supplying/bundling an installation medium:
+i.e. a USB key pre-loaded with our installer, with BIOS settings adjusted accordingly.
+Empowering the end-user with bare-metal re-install capability; assisted or otherwise.
```

Comment:
This will empower the designated end-user with bare-metal re-install capability, assisted or otherwise.